### PR TITLE
Updated the symfony console version

### PR DESCRIPTION
**Pull request by netesy**
updated to work with symfony 5 upwards

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     }],
     "require": {
         "league/climate": "^3.0",
-        "symfony/console": "^3.4|^4.2",
+        "symfony/console": "^5.0",
```
Comment (duncan3dc) [inline composer.json:16]: Why has support for 3.4 and 4.2 been dropped? Are Symfony no longer providing support for the 3.x or 4.x series?

Comment (netesy) [reply composer.json:16]: Honestly i don't know the reason i just noticed this error this morning.
`  - Conclusion: don't install symfony/console v4.4.6|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev
    - Conclusion: don't install symfony/console v4.4.5|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.4|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.6|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.3|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.5|install symfony/console v5.0.6|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.2|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.4|install symfony/console v5.0.5|install symfony/console v5.0.6|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.1|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.3|install symfony/console v5.0.4|install symfony/console v5.0.5|install symfony/console v5.0.6|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.0|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.2|install symfony/console v5.0.3|install symfony/console v5.0.4|install symfony/console v5.0.5|install symfony/console v5.0.6|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.0-RC1|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.1|install symfony/console v5.0.2|install symfony/console v5.0.3|install symfony/console v5.0.4|install symfony/console v5.0.5|install symfony/console v5.0.6|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.0-BETA2|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.0|install symfony/console v5.0.1|install symfony/console v5.0.2|install symfony/console v5.0.3|install symfony/console v5.0.4|install symfony/console v5.0.5|install symfony/console v5.0.6|install symfony/console v5.0.7
    - Conclusion: don't install symfony/console v4.4.0-BETA1|don't install illuminate/database 7.x-dev|keep symfony/console 5.1.x-dev|install symfony/console 5.0.x-dev|install symfony/console 5.1.x-dev|install symfony/console v5.0.0|install symfony/console v5.0.0-RC1|install symfony/console v5.0.1|install symfony/console v5.0.2|install symfony/console v5.0.3|install symfony/console v5.0.4|install symfony/console v5.0.5|install symfony/console v5.0.6|install symfony/console v5.0.7
    - illuminate/database 7.x-dev requires symfony/console ^5.0 -> satisfiable by symfony/console[5.1.x-dev, 5.0.x-dev, v5.0.0, v5.0.0-BETA1, v5.0.0-BETA2, v5.0.0-RC1, v5.0.1, v5.0.2, v5.0.3, v5.0.4, v5.0.5, v5.0.6, v5.0.7].
    - netesy/voom dev-master requires illuminate/database 7.x-dev -> satisfiable by illuminate/database[7.x-dev].
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|remove symfony/console 5.1.x-dev
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|remove illuminate/database 7.x-dev
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console 5.1.x-dev
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console 5.0.x-dev
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.7
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.6
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.5
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.4
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.3
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.2
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.1
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.0
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.0-RC1
    - Conclusion: don't install duncan3dc/symfony-climate 0.3.0|don't install symfony/console v5.0.0-BETA2
    - Conclusion: install duncan3dc/symfony-climate 0.3.0`

Comment (duncan3dc) [reply composer.json:16]: OK let me take a look at adding support for 5.0 alongside the existing supported versions...

Comment (netesy) [reply composer.json:16]: Thanks also consider adding the said support on your console project.

Comment (duncan3dc) [reply composer.json:16]: Support for Symfony 5.0 is available in [1.0.0](https://github.com/duncan3dc/symfony-climate/releases/tag/1.0.0) now, thanks for the prompt!

**Pull request closed without merging**